configure.ac

```diff
@@ -1506,6 +1506,28 @@
             [[#include <linux/net_tstamp.h>]])
     ])
 
+  # DPDK support
+    AC_ARG_ENABLE(dpdk,
+            AS_HELP_STRING([--enable-dpdk], [Enable DPDK support [default=no]]),
+                        [enable_dpdk=$enableval],[enable_dpdk=no])
+    AS_IF([test "x$enable_dpdk" = "xyes"], [
+        AC_DEFINE([HAVE_DPDK],[1],(DPDK support enabled))
+        PKG_CHECK_MODULES([libdpdk], libdpdk,, [with_pkgconfig_libdpdk=no])
+        if test "$with_pkgconfig_libdpdk" = "no"; then
+            echo
+            echo "   ERROR! libdpdk not found by pkg-config, go get it"
+            echo "   from https://www.dpdk.org/ or your distribution:"
+            echo
+            echo "   Ubuntu: apt-get install dpdk-dev"
+            echo "   Fedora: dnf install dpdk-devel"
+            echo "   CentOS/RHEL: yum install dpdk-devel"
+            echo
+            exit 1
+        fi
+        CFLAGS="${CFLAGS} ${libdpdk_CFLAGS}"
```

Comment:
no, that is pretty old way as a hack I had added for `https://github.com/vipinpv85/DPDK-Suricata_3.0`. This will conflict based on distro DPDK releases. There much easier way using configure and cmake to do this. please rework

Reply:
I reviewed your solutions of configure.ac in both of your Suricata branches but none seemed clean to me. 
However, I have reworked the process of getting the CFLAGS / LIBS..